go.mod

```diff
@@ -634,3 +636,5 @@ exclude (
 )
 
 replace github.com/github/smimesign => github.com/grafana/smimesign v0.2.1-0.20220408144937-2a5adf3481d3
+
+replace github.com/iamseth/oracledb_exporter => github.com/observiq/oracledb_exporter v0.0.0-20230123144034-6896822027ad
```

Comment:
nit (sorry!): could we move this to `replace` section where all other exporters live (line 589).

Reply:
no problem! updated 👍